src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -64,6 +64,12 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
             Command command = quizParser.parseCommand(commandText);
             commandResult = command.execute(model);
 
+            try {
```

Comment:
I think there is a problem with saving the addressbook this way. Try doing the quiz once and closing the application then run the application again and type a command e.g. "listnote" exit the application and all quizResults will be erased.

Reply:
Oh yea right. I will try to fix that.